eve/workers/master/bucketclient_packages.list

```diff
@@ -0,0 +1 @@
+nodejs
```

Comment:
If we only have one package for bucketclient I recommend putting `nodejs` directly into the dockerfile. What do you think?

Reply:
No prob for me it was just to keep the same format everywhere

Reply:
I think it's not a bad idea to try to stay consistent across ours projects

Reply:
It's a valid argument. Let's keep it that way. I think I did the same with S3